agent/canvas.py

```
@@ -659,41 +659,79 @@ def _node_finished(cpn_obj):
+                        def _schedule_tts(text):
+                            if tts_mdl and text:
+                                tts_tasks.append(asyncio.create_task(asyncio.to_thread(self.tts, tts_mdl, text)))
+
+                        async def _drain_ready_tts():
+                            events = []
+                            while tts_tasks and tts_tasks[0].done():
+                                task = tts_tasks.pop(0)
+                                try:
+                                    audio_binary = task.result()
+                                except Exception:
+                                    continue
```

Ruff (0.16.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)